internal/support/notifications/rest_notification.go

```diff
@@ -64,21 +64,19 @@ func notificationHandler(
 		return
 	}
 
-	if n.Severity == models.NotificationsSeverity(models.Critical) {
-		lc.Info("Critical severity scheduler is triggered for: " + n.Slug)
-		n, err = dbClient.GetNotificationById(n.ID)
-		if err != nil {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			lc.Error(err.Error())
-			return
-		}
+	lc.Info("The scheduler is triggered for: " + n.Slug)
```

Comment:
Recommend this LogLevel set at `Debug` rather than `Info`. Same for the completion message below. Given the default LogLevel of `Info` in configuration every notification will result in these two messages being written, resulting in chatty logging.

Reply:
Good idea 👍 Implemented, also rebased before pushing to remote repo.